### ISSUE TITLE
Localized string key in generated .editorconfig

### ISSUE THREAD

**Issue by 20chan** [dotnet/roslyn#39602]
**Version Used**: 

Visual Studio 2019 16.3.6, Korean, C#

**Steps to Reproduce**:

1. Set language to Korean
2. Generate .editorconfig from Tools->Options

**Expected Behavior**:

Only localized strings in comments.

**Actual Behavior**:

Key and value contains localized strings.

```editorconfig
#### 명명 스타일 ####

# 명명 규칙

dotnet_naming_rule.interface_should_be_i로_시작.severity = suggestion
dotnet_naming_rule.interface_should_be_i로_시작.symbols = interface
dotnet_naming_rule.interface_should_be_i로_시작.style = i로_시작
```

full .editorconfig: [gist](https://gist.github.com/20chan/a5bcb788d64769fe9a5eb1cdec3b011e)

**Comment by 20chan** [comment]
It happens only in naming style category.

https://github.com/dotnet/roslyn/blob/7d97db435f1d30285efdc8f77e1d573ea8b5075a/src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs#L32-L35

It seems like there is no `naming styles` resource.

**Comment by 20chan** [comment]
I found what was problem.

https://github.com/dotnet/roslyn/blob/a1533f06c8578787cd4ec607251607f6efab2468/src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigFileGenerator.cs#L93-L94

Here gets options by local language. And it's serializedNameMap was used into raw string.

https://github.com/dotnet/roslyn/blob/a1533f06c8578787cd4ec607251607f6efab2468/src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigFileGenerator.cs#L123-L125

**Comment by 20chan** [comment]
I submitted a PR [here](https://github.com/dotnet/roslyn/issues/39602).

**Comment by sharwell** [comment]
The localized strings appearing in keys and values are taken from user-modifiable content in the UI. Even if we account for all the default cases, it will be impossible to account for cases where the user has modified options prior to generating the .editorconfig file.

**Comment by sharwell** [comment]
Here is the UI where names can be changed by users:

![image](https://user-images.githubusercontent.com/1408396/68142816-02d52600-fee5-11e9-82c7-c20411802011.png)

**Comment by 20chan** [comment]
![이미지 004](https://user-images.githubusercontent.com/16171816/68170679-20bd8c00-ffb4-11e9-9dc2-62e18d49a74a.png)

Right, it was because just default generated keys are in localized language and also sample identifiers are.
But it just makes me confused. I had no idea why it is also localized when I generated .editorconfig fire at first, I didn't change anything at all. Default-generated localized .editorconfig seems like rude for other global programmers.

All codes-stuffs are my mistakes, I closed my PR.
And topic of this issue should be changed or something else.
